integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -0,0 +1,220 @@
+import json
+import logging
+import os
+import time
+import uuid
+from unittest import TestCase, mock
+
+import boto3
+
+import helpers
+
+
+class TestMultipleGranulesHappyPath(TestCase):
+    
+    def test_multiple_granules_happy_path(self):
+        self.maxDiff = None
+        """
+        If multiple granules are provided, should store them in DB as well as in recovery bucket.
+        """
+        try:
+            # Set up Orca API resources
+            # ---
+            my_session = helpers.create_session()
+            granule_id_1 = uuid.uuid4().__str__()
+            granule_id_2 = uuid.uuid4().__str__()
+            provider_id = uuid.uuid4().__str__()
+            provider_name = uuid.uuid4().__str__()
+            createdAt_time = int((time.time() + 5) * 1000)
+            collection_name = uuid.uuid4().__str__()
+            collection_version = uuid.uuid4().__str__()
+            bucket_name = "orca-sandbox-s3-provider"  # standard bucket where initial file exists
+            recovery_bucket_name = helpers.recovery_bucket_name
+            excluded_filetype = []
+            key_name_1 = "PODAAC/SWOT/ancillary_data_input_forcing_ECCO_V4r4.tar.gz"
```

Comment:
The file is 200 GB. Meets the large-file size requirement.

Reply:
sounds good. Didn't realize it was already 191GB lol